tests/behat/theme_boost_union_accessibilitysettings_support.feature

```diff
@@ -236,6 +236,25 @@ Feature: Configuring the theme_boost_union plugin for the "Support page" tab on
     Then I should see "Your accessibility support request was sent"
     # Unfortunately, the content of the email can't be tested with Behat yet as we do not have a way to test sent emails yet.
 
+  @javascript
+  Scenario: Support form success notification is shown before content-upper blocks on the homepage
```

Comment:
The test is perfecly fine and it's great that you re-used Boost Union's accessibility support form (instead of Moodle core's support form) to trigger the notification on site home.

I would just move the scenario to theme_boost_union_feelsettings_blocks.feature as its goal is to test the behaviour of blocks and not the support form.

I will move that with a review changes commit.